README.md

```diff
@@ -14,22 +14,23 @@ Current Features:
 
 In Progress:
 * Map Engine Tests Screen
+* Credits Screen
+* New Hero Screen
 
 Todo:
-* Credits Screen (Can be done now)
-* New Hero Screen (Can be done now except for text input)
 * Gameplay (when the engine supports it)
 
 # Installation
 
 1. Download OpenDiablo2 to a folder (or clone the repo)
 1. Copy the data files over from the copy of Diablo II you have. You need to do at least one of the following:
-   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the MPQs folder of this project (there should be 11 MPQs).
+   1. If you have good old Diablo II (latest LOD patch required), copy the MPQs from the game's installation folder into the DATA\MPQ-D2 folder of this project (there should be 11 MPQs).
```

Comment:
can you use / ?
\ implies windows and works only on windows, but / works everywhere, including windows